### PR TITLE
FINERACT-838 Swagger Codegen Integration

### PR DESCRIPTION
**Pull request by Grandolf49**
## Description
Integrated support for building API Client from OpenAPI Specification file. [FINERACT-838](https://issues.apache.org/jira/browse/FINERACT-838)

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [x] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [x] API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm has been updated with details of any API changes.

- [x] Integration tests have been created/updated for verifying the changes made.

- [x] All Integrations tests are passing with the new commits.

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

File: fineract-provider/build.gradle [hunk 1]
```diff
@@ -55,7 +55,8 @@ buildscript {
         classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.2"
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:1.2.1"
         classpath "com.diffplug.spotless:spotless-plugin-gradle:5.1.0"
-        classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.4"
+        classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.3"
```
Comment (thesmallstar) [inline fineract-provider/build.gradle:58]: Are you sure about this change? Does the newer version bring some issues?

**Comment by thesmallstar** [comment]
You need to do `./gradlew spotlessApply` for the build to pass :D

**Comment by Grandolf49** [comment]
Closing this PR since it contains inconsistent commit messages and the build is failing too

File: fineract-provider/build.gradle (hunk 1 above)
Comment (Grandolf49) [reply fineract-provider/build.gradle:58]: Hi @thesmallstar Yes, you're right, this change was not necessary. I have created a new PR for this. Please review #1206

**Pull request closed without merging**